packages/label-sync/src/label-sync.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import { labels } from './labels.json';
+import {request} from 'gaxios';
+import { GitHubAPI } from 'probot/lib/github';
+
+interface Repos {
+  repos: [{
+    language: string,
+    repo: string
+  }]
+}
+
+export = (app: Application) => {
+  const events = [
+    'repository.created',
```

Comment:
this seems reasonable to me 👍 

So if I edit labels they immediately go back to where they started.

The one use-case we don't cover, is how we periodically sync labels as new ones are added.

Reply:
We don't really need periodic :) Below we trigger a full on update whenever there are pushes to the repo-automation bots repo (with a todo to make it more scoped).